pkg/oc/oc.go

```diff
@@ -117,3 +120,65 @@ func CreateSecretsForTektonResults(){
 	//creating secret with generated certificate
 	cmd.MustSucceed("oc", "create", "secret", "tls", "-n", "openshift-pipelines", "tekton-results-tls", "--cert=cert.pem", "--key=key.pem")
 }
+
+func EnsureResutsReady(){
```

Comment:
@VeereshAradhya FYI maybe we should refactor our packages a bit. I would expect `pkg/oc/oc.go` to contain helper functions for `oc` not bunch of functions using `oc`.